anndata/tests/test_readwrite.py

```diff
@@ -631,6 +631,21 @@ def test_zarr_chunk_X(tmp_path):
     from_zarr = ad.read_zarr(zarr_pth)
     assert_equal(from_zarr, adata)
 
+def test_zarr_chunk_X_sparse(tmp_path):
+    import zarr
+
+    zarr_pth = Path(tmp_path) / "test.zarr"
+    adata = gen_adata((100, 100), X_type=csc_matrix)
+    adata.write_zarr(zarr_pth, chunks=(20,))
+
+    z = zarr.open(str(zarr_pth))  # As of v2.3.2 zarr won’t take a Path
+    assert z["X"]["data"].chunks == (20,)
+    assert z["X"]["indices"].chunks == (20,)
+    # Chunks arg only affects the "data" arrays.
```

Comment:
If you only wanted to access a subset of genes or cells, I would imagine you would run into similar problems if the `indptr` array was unchunked. Does it really make sense for chunking to not apply here?

Reply:
I was a little conflicted on this, but I suppose that it couldn't hurt like if you have a million variables on the row or column that `indptr` is for.  This array is probably smaller than `data` or `indices` anyway.